src/main/java/ch/njol/skript/ScriptLoader.java

```diff
@@ -452,7 +493,15 @@ private final static ScriptInfo loadScript(final @Nullable Config config) {
 			currentAliases.clear();
 			currentOptions.clear();
 			currentScript = config;
-			
+
+			/*
+			 * If editing this class, please remember to call this event
+			 * after currentScript has already been set to the provided Config,
+			 * but before the script actually starts parsing
+			 */
+			if (callPreLoadEvent)

```

Comment:
Would it be useful to expose *some* information in the pre-load event?

Reply:
like? The config object gives quite a bit of info

Reply:
The event does not reference the config.

Reply:
ScriptLoader#currentScript?

Reply:
added